packages/client/components/jobs/JobFilter.tsx

```diff
@@ -1,19 +1,146 @@
-import { Box, Container, Input } from '@chakra-ui/react';
+import { Plus, X } from 'tabler-icons-react';
 import { useTranslation } from 'next-i18next';
+import {
+  Box,
+  Button,
+  Container,
+  Input,
+  Text,
+  useControllableState,
+  Heading,
+  VStack,
+  StackDivider,
+} from '@chakra-ui/react';
+
+type filterLabelProps = {
+  text: string;
+};
```

Comment:
This is a nitpick. But please do prefer to write type declarations in pascal casing. 

This can easily be mistaken as a variable as per common conventions in JavaScript `camelCasing` is being used mostly for variables

Suggested change is to PascalCase this 

```ts
type FilterLabelProps = {
  text: string;
}
```

Reply:
Thank you @carlomigueldy I should be more familiar with the conventions :)